app/assets/stylesheets/question_index.scss

```diff
@@ -0,0 +1,25 @@
+.question-body {
+    font-size: larger;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Properties should be ordered font-family, font-size